### DISCUSSION TITLE
Migrate Laravel Mix with Vite

### DISCUSSION THREAD

**Discussion by ganjarsetia** [sujit-baniya/fiber-boilerplate#39] [Ideas]
Vite is a build tool that aims to provide a faster and leaner development experience for modern Front end web projects. I think is good idea to replace Mix with Vite. Here guide to migrate:

https://github.com/laravel/vite-plugin/blob/main/UPGRADE.md

What do you think?

**Comment by sujit-baniya** [comment]
@ganjarsetia Yeah it's good idea. I'll work on it

**Reply by ganjarsetia** [reply]
Nice!

I have another Ideas.

1. migrate Vue to 3
2. use [Go Embed](https://pkg.go.dev/embed) for embedding front end files

What do you think?

**Reply by sujit-baniya** [reply]
@ganjarsetia Yes we could do it. It might take some time for me to implement those changes. So I'll let you know once done. Or if you could make changes with PR, I would really appreciate

**Reply by ganjarsetia** [reply]
Thanks.

I'll try to create PR for migrating Vue to 3 & embed frontend files, but I need to wait for Vite first

Don't worry, there is no deadline or when it need to be done. You are the hero man!
